contrib/containers/guix/scripts/guix-start

```diff
@@ -2,19 +2,30 @@
 
 set -eo pipefail
 
+if [[ -n "${1}" ]]; then
+    WORKSPACE_PATH="$1"
+else
+    WORKSPACE_PATH="/src/dash"
+fi
+
+if [[ ! -d "$WORKSPACE_PATH" ]]; then
+    echo "$0: $WORKSPACE_PATH is not a valid directory, exiting!"
+    exit 1
+fi
+
 XCODE_VERSION="12.1"
 XCODE_RELEASE="12A7403"
 XCODE_ARCHIVE="Xcode-${XCODE_VERSION}-${XCODE_RELEASE}-extracted-SDK-with-libcxx-headers"
 
 # Check if macOS SDK is present, if not, download it
-if [ ! -d "/src/dash/depends/SDKs/${XCODE_ARCHIVE}" ]
+if [ ! -d "${WORKSPACE_PATH}/depends/SDKs/${XCODE_ARCHIVE}" ]
 then
-    mkdir -p /src/dash/depends/SDKs
-    curl -L https://bitcoincore.org/depends-sources/sdks/${XCODE_ARCHIVE}.tar.gz | tar -xz -C /src/dash/depends/SDKs
+    mkdir -p "${WORKSPACE_PATH}/depends/SDKs"
+    curl -L https://bitcoincore.org/depends-sources/sdks/${XCODE_ARCHIVE}.tar.gz | tar -xz -C "${WORKSPACE_PATH}/depends/SDKs"
 fi
 
-cd /src/dash
+cd ${WORKSPACE_PATH}
 git status >> /dev/null
-git config --global --add safe.directory /src/dash
+git config --global --add safe.directory ${WORKSPACE_PATH}
```

Comment:
(for some reason gh isn't letting me put a suggestion, but can you put the safe.directory above the git status? I needed that diff in order to make it work. Probably should've been fixed earlier